contents/tutorials/redirect-testing.md

```diff
@@ -0,0 +1,383 @@
+---
+title: How to do redirect testing
+date: 2023-10-19
+author: ["ian-vanagas"]
+showTitle: true
+sidebar: Docs
+featuredImage: ../images/tutorials/banners/tutorial-16.png
+tags: ['experimentation', 'feature flags']
+---
+
+Redirect testing is a way to A/B test web pages by redirecting users to one or the other. It is a popular way to split test landing pages, for example.
```

Comment:
The second sentence doesnt add anything I feel. Feels like it's just there for SEO keywords
```suggestion
Redirect testing is a way to A/B test web pages by redirecting users to one or the other.
```